thirdparty/tweetnacl/nacl-util.js

```diff
@@ -0,0 +1,81 @@
+// Written in 2014-2016 by Dmitry Chestnykh and Devi Mandiri.
```

Comment:
As this is from a different GitHub repository it should also be in a different folder.

Reply:
Ok. What do you think about your other third party scripts, which are located in modules? I'd also move them to the new third party folder, I just didn't do it for minimum code changes.

Reply:
I leave that up to you, as you are the one still working with that code. At that version I did not do any real separation on folder level between my code and unchanged included libraries.

My comment is only about if you start that separation, and have a folder `tweetnacl` there the readme in `thirdparty` states code from a certain repository is included, it also should only contain that code.

Note that if you decide to clean up also the exiting files and move them, the `rsasign-1.2.js` is not the original file. It is slightly modified by me.

Reply:
You did not move this file but added another copy. `thirdparty/tweetnacl/nacl-util.js` should get deleted.

Reply:
My error, I've removed the file. It wasn't even used anymore...